### ISSUE TITLE
MDNS doesn't work on built win32 app

### ISSUE THREAD

**Issue by timmo001** [timmo001/system-bridge#45]
# Description

```bash
warn: MDNS error: \\?\C:\Users\...\AppData\Local\Temp\...GUID....tmp.node is not a valid Win32 application.
````

**Comment by timmo001** [comment]
Bump

**Comment by SilentException** [comment]
Trying out v1.7.0 this is the last entry in the log before the app terminates on start. v1.6.0 started fine despite the same log entry. Might be something different but it is not logged. Using Windows 10.

**Comment by timmo001** [comment]
This might be something that cannot be done without the ability to install the bonjour sdk for windows on a build server, so may need to go stale

**Comment by SilentException** [comment]
@timmo001 I don't mind the error in the log, v1.6.0 had it and it was was/is working. Still need to try latest version v1.7.2 but this is the last log entry before 1.7.0 terminates so that's why I commented (since I'm not sure why 1.7.0 was crashing.

**Comment by timmo001** [comment]
Fixed in #432
